.github/workflows/release.yml

```diff
@@ -0,0 +1,47 @@
+name: release
+on:
+  workflow_dispatch:
+    # Inputs the workflow accepts.
+    inputs:
+      tag:
+        description: 'tag and release to create'
+        # Input has to be provided for the workflow to run
+        required: true
+env:
+  GITHUB_ENV: ".env"
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
+      - name: run make bump-version
+        run: make bump-version
+        env:
+          NEW_VERSION: ${{ github.event.inputs.tag }}
+      - name: commit manifests
+        uses: EndBug/add-and-commit@a988073222b8bd50f3ecfca9c0ab7dfbf0d08ceb
+        with:
+          message: "create release manifests for ${{ github.event.inputs.tag }}"
+          add: "releases/"
```

Comment:
The `bump-version` Makefile target [modifies a lot of files across the repo](https://github.com/digitalocean/digitalocean-cloud-controller-manager/blob/a843046121562eec208cc12c9637ada7894e4ef7/Makefile#L51-L58) in an attempt to increase all mentioned "latest version" specifiers. This was always pretty gross IMO, and has occasionally targeted files that shouldn't have been updated. If we automate the commit, then I think we should first have a PR that removes these latest version specifiers so that we only commit the `releases/` directory and the `VERSION` file.

Reply:
bump version alters the version in these files:
- VERSION
- README.md
- docs/getting-started.md
- docs/example-manifests/cloud-controller-manager.yml
- releases/${NEW_VERSION}.yml

These are valid changes IMO and we should commit them. I'll modify this task to add them as well.

Reply:
What I was trying to say is that I don't like how we have been doing these mass replacements across several documentation files just to have the latest released version mentioned at several convenient places. This was somewhat acceptable when we did the release manually, but we shouldn't let automation do this. As I wrote, there have been situations in the past where those replacements changed portions of the repo that shouldn't have been changed, it was only due to the interactive release process that we could undo them before they were pushed to the git server.

Customers/users can figure out what the latest version is by looking at the releases page, so my strong preference would be to do a quick PR that removes the latest references from the docs before merging this one.

Reply:
Only added releases/ and VERSION file here. Will follow up with the bump versions change in the next PR